subsys/bluetooth/common/Kconfig

```diff
@@ -60,37 +64,20 @@ config BT_BUF_ACL_RX_SIZE
 	help
 	  Maximum support ACL size of data packets sent from the Controller to
 	  the Host. This value does not include the HCI ACL header.
-	  In a combined Host and Controller build the buffer sizes in both the
-	  Host and the Controller will use this value for buffer sizes, and
-	  therefore Controller to Host flow Controller is not needed.
+
+	  In a combined Host and Controller build, both the
+	  Host and the Controller will use this value for buffer sizes, making
+	  Controller to Host flow control not strictly necessary.
```

Comment:
It is not true that host flow control has no effect just because the host and controller have the same number of ACL buffers. I'm not sure what this sentence is getting at. It's too vague.

Reply:
it's yet another optimization by the original devs. that's why flow control is disabled on single-chip builds